.github/workflows/_test.yml

```diff
@@ -5,24 +5,50 @@ on:
 permissions: {}
 
 jobs:
+  lint-test:
+    name: Lint Test
+    runs-on: ubuntu-latest
+
+    env:
+      # Set TOXENV env var to tell tox which testenv (see tox.ini) to use
+      TOXENV: lint
```

Comment:
You can also configure toxenv via command line argument, then you don't need to set an envvar.
```suggestion
```